doc/recipes/migrating-database-versions.md

```diff
@@ -0,0 +1,64 @@
+## Migrating database versions
+
+Database migrations are done using [Room](https://developer.android.com/training/data-storage/room/migrating-db-versions). These migrations are run 
+when the app is opened for the first time after an update. 
```

Comment:
there's an extra space here, can you format the file, please?

Reply:
At the end of the line?

Reply:
I pushed the change. PTAL